experimental/uefi/loader/src/main.rs

```diff
@@ -65,11 +65,22 @@ struct CommsChannel {
     inner: UnixStream,
 }
 
-impl Channel for CommsChannel {
-    fn send(&mut self, data: &[u8]) -> anyhow::Result<()> {
+impl ciborium_io::Write for CommsChannel {
```

Comment:
We should probably just always import them and use them unqualified, but up to you.